CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 0.102.0
+
+- Previously, the `--resolve-bugzillas` option has been used for `create-update` and the option `--resolve-bug` has been used for `propose-downstream`. They have been unifed to just `--resolve-bug` for better user experience.
+  If you used the option `--resolve-bugzillas` for command `create-update` you must use the `--resolve-bug` now instead.
+
+Previously, the `pull-from-upstream` option took the `--resolve-bug` option from CLI.
+While the `resolved-bugs` option has been used for retriggering an comment command in service. . They have been unifed to just `--resolve-bug` for better user experience.
+If you used the option `--resolved-bugs` for retriggering comment command you must use the `--resolve-bug` now instead. (#2428)
+
```

Comment:
```suggestion
- Previously, the `--resolve-bugzillas` option has been used for `create-update` and the option `--resolve-bug` has been used for `propose-downstream`. They have been unified to just `--resolve-bug` for better user experience.
  If you used the option `--resolve-bugzillas` for command `create-update` you must use the `--resolve-bug` (or shorthand `-b`) now instead.

  Previously, the `pull-from-upstream` option took the `--resolve-bug` option from CLI.
  While the `resolved-bugs` option has been used for retriggering an comment command in service. They have been unified to just `--resolve-bug` for better user experience.
  If you used the option `--resolved-bugs` for retriggering comment command you must use the `--resolve-bug` (or shorthand `-b`) now instead. (#2428)

```

Reply:
I left out the service part and reworded the rest a bit.

Reply:
thanks, I will have a look at the argparse and bump it up a bit… it's beginning to be annoying